babel.config.js

```diff
@@ -1,4 +1,11 @@
 module.exports = {
-  extends: './packages/.babelrc',
-  presets: [['@babel/env', {'modules': 'commonjs'}], '@babel/react'],
-};
+  presets: [
```

Comment:
What are these babel configs used for?

Reply:
Jest still uses `babel-jest`. Future work is to possibly use `ts-jest` to typecheck tests.